cmd/delete/delete.go

```diff
@@ -41,7 +50,12 @@ delete all files bigger than 100MBytes.
 		cmd.CheckArgs(1, 1, command, args)
 		fsrc := cmd.NewFsSrc(args)
 		cmd.Run(true, false, command, func() error {
-			return operations.Delete(context.Background(), fsrc)
+			err := operations.Delete(context.Background(), fsrc)
```

Comment:
More idiomatic go here would be to take an early return so

```go
err := operations.Delete(context.Background(), fsrc)
if err != nil {
    return err
}
fdst := cmd.NewFsDir(args)
return operations.Rmdirs(context.Background(), fdst, "", true)
```

Reply:
Sure, I will do an early return as suggested. Sorry but in your code you have also removed the rmdirs flag check, was that on purpose? How else should we check for that?
```
if err := operations.Delete(context.Background(), fsrc); err != nil {
	return err
}
if rmdirs {
	fdst := cmd.NewFsDir(args)
	return operations.Rmdirs(context.Background(), fdst, "", true)
}
return nil
```

Reply:
Yes that looks fine now